infrastructure/frontend/apply-frontend-infrastructure.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I'm assuming this is solely for development because higher environment deployments will be done using the pipeline stacks? Am I right?

Reply:
Correct, this is for manual deployment at least for now (though we might be able to resuse the scripts in the workflows and ajust them as needed)